doc/oper.xml

```diff
@@ -962,6 +962,96 @@ gap> DigraphJoin(gr, gr2);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DigraphCartesianProduct">
+<ManSection>
+  <Func Name="DigraphCartesianProduct" Arg="gr1, gr2, ..." Label="for an arbitrary 
+   number of digraphs"/>
+  <Func Name="DigraphCartesianProduct" Arg="list" Label="for a list of digraphs"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    In the first form, if <A>gr1</A>, <A>gr2</A>, etc. are digraphs, then
+    <C>DigraphCartesianProduct</C> returns their Cartesian product.
+
+    In the second form, if <A>list</A> is a non-empty list of digraphs,
+    then <C>DigraphCartesianProduct</C> returns the Cartesian product of 
+    the digraphs contained in the list. <P/>
+
+    The Cartesian product of two digraphs <C>G</C>, <C>H</C> is a digraph with 
+    vertex set <C>Cartesian(DigraphVertices(G), DigraphVertices(H))</C> and there
```

Comment:
Mathematically, fine, but since the vertex set of a digraph is a list of positive integers in the Digraphs package, you also need to say how this mathematical definition actually relates to what happens in the package.